test/auth/registerDetails.mother.ts

```diff
@@ -5,6 +5,8 @@ export function emptyRegisterDetails(): RegisterDetails {
     privateKeyEncrypted: '',
     publicKey: '',
     revocationCertificate: '',
+    publicKyberKey: '',
```

Comment:
Same here with the `keys` field

Reply:
Done. Though this leads to a bit cumbersome field accesses `keys.keys.ecc.privateKeyEncrypted`. Maybe we can drop `keys` here and use `ecc` and `kyber` directly?